fire/core_test.py

```diff
@@ -19,13 +19,36 @@
 from fire import core
 from fire import test_components as tc
 from fire import trace
+import contextlib
 import mock
-
+import re
+import six
+import sys
 import unittest
 
 
-class CoreTest(unittest.TestCase):
+class BaseTest(unittest.TestCase):
+  @contextlib.contextmanager
+  def assertRaisesFireExit(self, code, regexp=None):
```

Comment:
could move this base class somewhere else - wasn't sure how you wanted to organize it.